frontend/packages/console-shared/src/utils/sample-utils.ts

```diff
@@ -0,0 +1,263 @@
+import { Map as ImmutableMap } from 'immutable';
+import {
+  BuildConfigModel,
+  ClusterRoleModel,
+  ConsoleLinkModel,
+  NetworkPolicyModel,
+  ResourceQuotaModel,
+  RoleModel,
+} from '@console/internal/models';
+import {
+  apiVersionForModel,
+  GroupVersionKind,
+  K8sKind,
+  K8sResourceKind,
+  referenceForModel,
+} from '@console/internal/module/k8s';
+
+import * as denyOtherNamespacesImg from '@console/internal/imgs/network-policy-samples/1-deny-other-namespaces.svg';
+import * as limitCertainAppImg from '@console/internal/imgs/network-policy-samples/2-limit-certain-apps.svg';
+import * as allowIngressImg from '@console/internal/imgs/network-policy-samples/3-allow-ingress.svg';
+import * as defaultDenyAllImg from '@console/internal/imgs/network-policy-samples/4-default-deny-all.svg';
+import * as webAllowExternalImg from '@console/internal/imgs/network-policy-samples/5-web-allow-external.svg';
+import * as webDbAllowAllNsImg from '@console/internal/imgs/network-policy-samples/6-web-db-allow-all-ns.svg';
+import * as webAllowProductionImg from '@console/internal/imgs/network-policy-samples/7-web-allow-production.svg';
+import { FirehoseResult } from '@console/internal/components/utils';
+import * as _ from 'lodash';
+
+export type Sample = {
+  highlightText?: string;
+  title: string;
+  img?: string;
+  description: string;
+  id: string;
+  yaml?: string;
+  snippet?: boolean;
+  targetResource: {
+    apiVersion: string;
+    kind: string;
+  };
+};
+
+const getTargetResource = (model: K8sKind) => ({
+  apiVersion: apiVersionForModel(model),
+  kind: model.kind,
+});
+
+const clusterRoleBindingSamples: Sample[] = [
+  {
+    title: 'Allow reading Nodes in the core API groups (for ClusterRoleBinding)',
```

Comment:
All these should be translated... but we can probably run through this as a bug if needed